README.md

```diff
@@ -1,67 +1,59 @@
+|[Assistant Core](/generators/generator-bot-assistant-core) | [![npm version](https://badge.fury.io/js/%40microsoft%2Fgenerator-bot-assistant-core.svg)](https://badge.fury.io/js/%40microsoft%2Fgenerator-bot-assistant-core) | The base assistant-style bot (no skills). |
+|[Enterprise Assistant](/generators/generator-bot-enterprise-assistant) | [![npm version](https://badge.fury.io/js/%40microsoft%2Fgenerator-bot-enterprise-assistant.svg)](https://badge.fury.io/js/%40microsoft%2Fgenerator-bot-enterprise-assistant) | Assistant Core + Calendar & People as skills. |
 
 ### Packages
 
-Bits of bots that you can add to your bot project.
+Bits of bots that you can add to your bot project. They can contain coded extensions like custom actions, adapters, or triggers, and declarative assets like dialogs, language generation or language understanding files.
 
-| Name         |Type   | Description |
-|:------------:|:------|-------------|
-|[Welcome](/packages/Welcome) | Dialogs | Declarative assets supporting scenarios that welcome new and returning users. |
-|[HelpAndCancel](/packages/HelpAndCancel) | Dialogs | Declarative assets supporting scenarios for "help" and "cancel" utterances. |
-|[Onboarding](/packages/onboarding) | Dialogs |Declarative assets supporting a first time user experience. |
-|[Calendar](/packages/Calendar) | Custom Actions |Custom actions supporting Calendar scenarios. |
-|[Graph](/packages/Graph) | Custom Actions |Custom actions for work with the MS Graph API|
-|[Orchestrator](Microsoft.Bot.Builder.AI.Orchestrator) | Recognizer | Plugin to register the Orchestrator recognizer with the runtime. |
+| Name         |Type   | NuGet | npm |Description |
+|:------------:|:------|:-----:|:---:|:-----------|
+|[Welcome](/packages/Welcome) | Dialogs | [![NuGet Badge](https://buildstats.info/nuget/Microsoft.Bot.Components.Welcome?includePreReleases=true)](https://www.nuget.org/packages/Microsoft.Bot.Components.Welcome/)| [![npm version](https://badge.fury.io/js/%40microsoft%2Fbot-components-welcome.svg)](https://badge.fury.io/js/%40microsoft%2Fbot-components-welcome) | Declarative assets supporting scenarios that welcome new and returning users. |
+|[HelpAndCancel](/packages/HelpAndCancel) | Dialogs | [![NuGet Badge](https://buildstats.info/nuget/Microsoft.Bot.Components.HelpAndCancel?includePreReleases=true)](https://www.nuget.org/packages/Microsoft.Bot.Components.HelpAndCancel/) | [![npm version](https://badge.fury.io/js/%40microsoft%2Fbot-components-helpandcancel.svg)](https://badge.fury.io/js/%40microsoft%2Fbot-components-helpandcancel) | Declarative assets supporting scenarios for "help" and "cancel" utterances. |
+|[Graph](/packages/Graph) | Custom Actions | [![NuGet Badge](https://buildstats.info/nuget/Microsoft.Bot.Components.Graph?includePreReleases=true)](https://www.nuget.org/packages/Microsoft.Bot.Components.Graph/) | Soon! | Custom actions for work with calendars and people through the MS Graph API.|
```

Comment:
I've found that giving estimates like "soon!" in repo docs can lead to false assumptions and more to cleanup later. If we're going to give indeterminate estimates we should at least include a file issues to track it so that users can monitor our progress.